Inc/DDSTextureLoader.h

```diff
@@ -47,6 +47,7 @@ namespace DirectX
             DDS_LOADER_DEFAULT = 0,
             DDS_LOADER_FORCE_SRGB = 0x1,
             DDS_LOADER_IGNORE_SRGB = 0x2,
+            DDS_LOADER_IGNORE_MIPS = 0x20,
```

Comment:
The values of these bits are intended to match *DirectX Tool Kit for DX12* to avoid issues when mixing them in the same app.

```
        enum DDS_LOADER_FLAGS : uint32_t
        {
            DDS_LOADER_DEFAULT = 0,
            DDS_LOADER_FORCE_SRGB = 0x1,
            DDS_LOADER_IGNORE_SRGB = 0x2,
            DDS_LOADER_MIP_AUTOGEN = 0x8,
            DDS_LOADER_MIP_RESERVE = 0x10,
            DDS_LOADER_IGNORE_MIPS = 0x20,
        };
```